### PR TITLE
Troubleshooting

### PR DESCRIPTION
**Pull request by mahebbar**

### CODE REVIEW

File: docs/tsg_collect_logs.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Logger is a Kubernetes job which can be run on a cluster to extract pod logs and cluster information. It helps in troubleshoot/debugging activities. Logger runs the logger container *openebs/logger* and it is recommended to run for a specific duration to capture logs while attempting to reproduce issues.
+
+### Benefits
+
+Following are the benefits of using Logger.
+
+- The purpose of Logger is to obtain debug-information/quick logs in clusters where more standard logging frameworks like EFK are not already configured. Logger creates a simple support bundle which can be provided to debug teams.
+- This may be the case with most "non-production/development" infrastructures.
+
```
Comment (ksatchit) [inline docs/tsg_collect_logs.md]: Let us add "Where is the logger found" - and point to the location where the kubernetes job is available (in openebs/test-storage)

File: docs/tsg_pvs.md [hunk 2]
```diff
@@ -72,49 +84,60 @@ This issue is due to failed application operations in the container. Typically t
...
   3. If any one of the replicas are still in RO mode, wait for the synchronization to complete. If all the replicas are in RO mode (this may occur when all replicas re-register into the controller within short intervals), you must restart the OpenEBS volume controller using the `kubectl delete pod <pvc-ctrl>`  command . Since it is a Kubernetes deployment, the controller pod is restarted successfully. Once done, verify that all replicas transition into *RW mode*.
 
-  4. Unmount the stale iscsi device mounts on the application node. Typically, these devices are mounted in the `/var/lib/kubelet/plugins/kubernetes.io/iscsi/iface-default/<target-portal:iqn>-lun-0`  path.
+  4. Un-mount the stale iscsi device mounts on the application node. Typically, these devices are mounted in the `/var/lib/kubelet/plugins/kubernetes.io/iscsi/iface-default/<target-portal:iqn>-lun-0`  path.
+
+  5. Identify whether the iSCSI session is re-established after failure. This can be verified using `iscsiadm -m session`, with the device mapping established using `iscsiadm -m session -P 3` and `fdisk -l`. **Note:**  Perform the following steps as part of the recovery procedure for a Volume-Read only issue.
```
Comment (ksatchit) [inline docs/tsg_pvs.md]: This kubelet-based recovery should replace the last "Note" (after pt 9) and not in pt 5. Let us retain the old pt.5 as is.

File: docs/tsg_collect_logs.md (hunk 1 above)
Comment (mahebbar) [reply docs/tsg_collect_logs.md]: Done

File: docs/tsg_pvs.md (hunk 2 above)
Comment (mahebbar) [reply docs/tsg_pvs.md]: Done!

File: docs/tsg_pvs.md [hunk 3]
```diff
@@ -72,49 +82,58 @@ This issue is due to failed application operations in the container. Typically t
...
 
-  9. Ensure that the application uses the new mount by forcing it to restart on the same node. This can be done using `docker stop <id>` of the application container on the node. Kubernetes will automatically restart the pod to ensure the "desirable" state.
+  9. Ensure that the application uses the newly mounted disk by forcing it to restart on the same node. Use the command  `docker stop <id>`  of the application container on the node. Kubernetes will automatically restart the pod to ensure the "desirable" state.
 
      While this step may not be necessary most times (as the application is already undergoing periodic restarts as part of the CrashLoop cycle), it can be performed if the application pod's next restart is scheduled with an exponential back-off delay.
 
-**Note:** In environments where the kubelet runs in a container (RancherOS, CoreOS, Containerized OpenShift deployment), you may not need perform the iSCSI re-login and remount steps explicitly. Instead, it is replaced by the kubelet container restart on the node.
+**Note:**  Perform the following steps as part of the recovery procedure for a Volume-Read only issue.
```
Comment (ksatchit) [inline docs/tsg_pvs.md]: **Note:**  In environments where the kubelet runs in a container, perform the following steps as part of the recovery procedure for a Volume-Read only issue.

**Pull request merged**